main.tf

```diff
@@ -0,0 +1,193 @@
+# TODO: (input) Create ACM cert outside of this module
+
+# TODO: use `terraform-terraform-label` to generate all labels.
+
+# TODO: (output) security group IDs
+
+#module "container_definition" {
+#  source    = "git::https://github.com/cloudposse/terraform-aws-ecs-container-definition.git?ref=init"
+#  container_name      = "app"
+#  container_image    = "nginx"
+#}
+
+# ECR repository
+resource "aws_ecr_repository" "app" {
```

Comment:
Let's move this out of this module. We provide https://github.com/cloudposse/terraform-aws-ecr